EveripediaNetwork/ep-ui
# New search functionality

# New search functionality

_This PR focuses on the improvement of the search bar on the navbar with a new UI, adding more content and separating results into different components i.e Wiki categories, events, accounts, etc._

## How should this be tested?

1. On the navbar, enter a search query, it should bring out results based on the search query
2. The dropdown for the result will have split the results into categories, click on the tabs to see if they are properly separated
3. Click on a result to redirect to a wiki page or accounts page
4. When clicked on a result it should save the result so when a user comes to search for another query will see recently searched wiki

## Figma link
https://www.figma.com/design/wosuJaHeV318k1ON82CZpo/%F0%9F%A7%A0-IQ.wiki?node-id=15327-102348&t=uhcxUfodI77Z1Dn2-11

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/2976
